components/kyma-environment-broker/internal/storage/driver/memory/operation.go

```diff
@@ -17,6 +17,7 @@ import (
 type operations struct {
 	mu sync.Mutex
 
+	operations               map[string]internal.Operation
```

Comment:
Map `operations` is not searched in `GetOperationByID` method. So you can insert, update operation (i.e. internal.Operation) but could not find it using this method.